erizo/src/erizo/NicerConnection.cpp

```diff
@@ -162,6 +162,9 @@ void NicerConnection::async(function<void(std::shared_ptr<NicerConnection>)> f)
 }
 
 void NicerConnection::start() {
+  ufrag_ = getNewUfrag();
```

Comment:
👍 I assume this is because you found it was possible to request the frag and pass before they were generated. Good catch.

Reply:
yep, that happens in the new negotiations, when Erizo sends the first offer.